Readme.md

```diff
@@ -1,59 +1,60 @@
+	CHECK_EQUAL_VAR(abs(-1.5), 1.5)
+	CHECK_EQUAL_VAR(abs(NaN), NaN)
+	WARN(abs(NaN) == NaN)
+	CHECK_EQUAL_VAR(abs(Inf), Inf)
+	CHECK_EQUAL_VAR(abs(-Inf), Inf)
+End
+```
 
-1. Install Igor
-2. Start Igor, this will create a folder called WaveMetrics in Documents
-3. Extract the zip file into the a folder, e. g. `thomas/unitTestingFramework` somewhere on your disc
-4. Create a link from `unitTestingFramework/procedures` to `Documents\WaveMetrics\Igor Pro 6 User Files\User Procedures`
-5. Have a look at the manual or the example experiments
+The framework itself combines numerous features:
 
-# Building the documentation
+* Test assertions for all kinds of conditions in three main flavours: Warnings, Checks and Requirements
+* Test cases can be grouped in test suites, run with multiple inputs, run in independent modules and contain background activity
+* Support of standard output formats like [JUNIT](https://junit.org/junit4) and [TAP](https://testanything.org/tap-version-13-specification.html)
+* Support for automated execution as part of a CI pipeline
+* Optional code coverage determination
+* Optional memory leak tracking
+* Fully documented and comes with lots of examples
+* Easily expandable and adaptable
 
-## Requirements
-The build process is fully automated using [docker containers](https://www.docker.com/). You will need
-- [docker](https://www.docker.com/get-docker)
-- [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+# Documentation
 
-## Building
+The [documentation](https://docs.byte-physics.de/igor-unit-testing-framework)
+contains a [guided tour](https://docs.byte-physics.de/igor-unit-testing-framework/guided-tour.html)
+and an [introduction](https://docs.byte-physics.de/igor-unit-testing-framework/basic.html) to the basic structure.
 
-Currently the automated build process only works on Linux.
+# Requirements
 
-Run `make` from the documentation root directory `docu/`:
-```bash
-cd docu
-make
-```
+Igor Pro version 6.37 or newer on Windows and MacOSX
 
-This will create a docker container with all the required dependencies and output the manual as pdf to `docu/manual.pdf` and html to the `docu/sphinx/html` subdirectory.
-The documentation is built using [doxygen](http://www.doxygen.org/), a [home-built awk script](https://github.com/byte-physics/doxygen-filter-ipf/), [breathe](https://github.com/michaeljones/breathe) and [sphinx](http://www.sphinx-doc.org).
+# Installation
 
-[The current documentation can be found on our website.](https://docs.byte-physics.de/igor-unit-testing-framework/)
+1. Extract the zip file into a folder, e. g. `alice/IgorTestingFramework`
+2. Open the Igor Pro user folder by selecting `Help/Show Igor Pro User Files` from the menu in Igor Pro
+3. Create a link from `alice/IgorTestingFramework/procedures` in the `User Procedures` subfolder
+4. Have a look at the [documentation](https://docs.byte-physics.de/igor-unit-testing-framework) or the example experiments
```

Comment:
This sounds a bit weird. Can you write something like: "Installation finished. You are ready to use IUTF in Igor Pro. Have a look..."?